core/capabilities/remote/target/caller.go

```diff
@@ -0,0 +1,141 @@
+package target
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+	commoncap "github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+	"github.com/smartcontractkit/chainlink/v2/core/capabilities/remote/target/request"
+	"github.com/smartcontractkit/chainlink/v2/core/capabilities/remote/types"
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+)
+
+type callerRequest interface {
```

Comment:
what is the significance of this interface? 
- why bother with an interface rather than a struct if it's private?
- what does it mean for a 'request' to have an 'OnMessage'? isn't a request a kind of message?

Reply:
yeah, no need to be an interface, left over from some refactor I guess, removed.